documentation/modules/ROOT/pages/crafting-cluster-telco-related-infra-operators-configs.adoc

```diff
@@ -298,7 +338,77 @@ spec:
   remediationAction: inform
   sourceFiles:
     - fileName: DisableSnoNetworkDiag.yaml
-      policyName: group-policies
+      policyName: "group-policy"
+    - fileName: DisableOLMPprof.yaml # wave 10
+      policyName: "group-policy"
+    - fileName: PerformanceProfile.yaml
+      policyName: "group-policy"
+      metadata:
+        annotations:
+          kubeletconfig.experimental: |
+            {"topologyManagerScope": "pod",
+             "systemReserved": {"memory": "3Gi"}
+            }
+      spec:
+        cpu:
+          isolated: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-cpu-isolated" (index .ManagedClusterLabels "hardware-type")) hub}}'
+          reserved: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-cpu-reserved" (index .ManagedClusterLabels "hardware-type")) hub}}'
+        hugepages:
+          defaultHugepagesSize: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-hugepages-default" (index .ManagedClusterLabels "hardware-type"))| hub}}'
+          pages:
+          - count: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-hugepages-count" (index .ManagedClusterLabels "hardware-type")) | toInt hub}}'
+            size: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-hugepages-size" (index .ManagedClusterLabels "hardware-type")) hub}}'
+        numa:
+          topologyPolicy: single-numa-node
+        realTimeKernel:
+          enabled: false
+        globallyDisableIrqLoadBalancing: false
+        # WorkloadHints defines the set of upper level flags for different type of workloads.
+        workloadHints:
+          realTime: true
+          highPowerConsumption: false
+          perPodPowerManagement: false
+    - fileName: TunedPerformancePatch.yaml
+      policyName: "group-policy"
+      spec:
+        profile:
+          - name: performance-patch
+            data: |
+              [main]
+              summary=Configuration changes profile inherited from performance created tuned
+              include=openshift-node-performance-openshift-node-performance-profile
+              # When using the standard (non-realtime) kernel, remove the kernel.timer_migration override from the [sysctl] section
+              [sysctl]
+              kernel.timer_migration=0
+              [scheduler]
+              group.ice-ptp=0:f:10:*:ice-ptp.*
+              group.ice-gnss=0:f:10:*:ice-gnss.*
+              [service]
+              service.stalld=start,enable
+              service.chronyd=stop,disable
+
+EOF
+cat <<EOF > ~/5g-deployment-lab/ztp-repository/site-policies/fleet/testing/group-hardware-types-configmap-test.yaml
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: group-hardware-types-configmap-test
```

Comment:
The ConfigMap name does not match the ConfigMap name used in the templates above: _group-hardware-types-configmap_